core/templates/dev/head/pages/exploration_editor/editor_tab/SolutionVerificationService.js

```diff
@@ -29,8 +29,9 @@ oppia.factory('SolutionVerificationService', [
         var rulesServiceName = (
           angularNameService.getNameOfInteractionRulesService(interactionId));
         var rulesService = $injector.get(rulesServiceName);
-        AnswerClassificationService.getMatchingClassificationResult(
-          explorationId, state, correctAnswer, true, rulesService
+        return (
+          AnswerClassificationService.getMatchingClassificationResult(
+          explorationId, state, correctAnswer, true, rulesService)
```

Comment:
Hm, I'm returning the promise so that it is easier to test (using then and done to check if the async verification was successful in the solutionVerificationServiceSpec).